Cargo.toml

```diff
@@ -3,7 +3,8 @@ authors = ["The Rust Project Developers"]
 description = "A collection of numeric types and traits for Rust, including bigint,\ncomplex, rational, range iterators, generic integers, and more!\n"
 documentation = "http://rust-num.github.io/num"
 homepage = "https://github.com/rust-num/num"
-keywords = ["mathematics", "numerics"]
+keywords = ["mathematics", "numerics", "bignum"]
+categories = [ "algorithms", "data-structures", "science", "cryptography" ]
```

Comment:
Thanks. Rebased, and I've added some suggestions to the subcrates. The [category slugs](https://crates.io/category_slugs) are a bit lacking, IMHO, and I also don't understand the purpose of all the subcrates, so again any suggestions are welcome!